src/main/java/com/byznass/tiolktrack/kernel/dao/LocationProvider.java

```diff
@@ -0,0 +1,11 @@
+package com.byznass.tiolktrack.kernel.dao;
+
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
+import com.byznass.tiolktrack.kernel.model.Location;
+
+import java.util.List;
+
+public interface LocationProvider {
+
+	List<Location> getLocationsForGps(String gpsId) throws TiolkTrackException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)